scripts/run-clang-format.py

```diff
@@ -22,7 +22,7 @@
 import sys
 import tempfile
 
-EXPECTED_VERSION = "10.0.0"
+EXPECTED_VERSION = "11.1.0"
```

Comment:
Noting this change because the version of `clang-format` in the gpuCI images is 11.1.0; are we okay to change this?

Reply:
Yes, that is perfectly fine